--- origsrc/geoclue-0.11.1/example/Makefile.am	2008-04-25 08:39:19.000000000 -0500
+++ src/geoclue-0.11.1/example/Makefile.am	2009-10-30 10:00:01.938178400 -0500
@@ -17,7 +17,7 @@ revgeocode_example_SOURCES = revgeocode-
 master_example_SOURCES = master-example.c
 master_pos_example_SOURCES = master-pos-example.c
 
-AM_CFLAGS = $(GEOCLUE_CFLAGS)
+AM_CFLAGS = -I$(top_srcdir) $(GEOCLUE_CFLAGS)
 LDADD = \
 	$(GEOCLUE_LIBS) \
     $(top_builddir)/geoclue/libgeoclue.la
--- origsrc/geoclue-0.11.1/test/Makefile.am	2008-04-25 08:39:19.000000000 -0500
+++ src/geoclue-0.11.1/test/Makefile.am	2009-10-30 10:01:12.132676800 -0500
@@ -8,6 +8,7 @@ geoclue_test_gui_LDADD = \
 	$(top_builddir)/geoclue/libgeoclue.la
 
 geoclue_test_gui_CFLAGS= \
+	-I$(top_srcdir) \
 	$(GTK_CFLAGS) \
 	$(GEOCLUE_CFLAGS)
 
