--- origsrc/geoclue-0.12/docs/reference/Makefile.am	2010-03-25 15:43:10.000000000 -0500
+++ src/geoclue-0.12/docs/reference/Makefile.am	2011-04-07 03:11:50.674066800 -0500
@@ -94,7 +94,7 @@ expand_content_files=
 # signals and properties.
 # e.g. INCLUDES=-I$(top_srcdir) -I$(top_builddir) $(GTK_DEBUG_FLAGS)
 # e.g. GTKDOC_LIBS=$(top_builddir)/gtk/$(gtktargetlib)
-INCLUDES=-I$(top_srcdir) $(GEOCLUE_CFLAGS)
+INCLUDES=-I$(top_builddir) -I$(top_srcdir) $(GEOCLUE_CFLAGS)
 GTKDOC_LIBS=$(top_builddir)/geoclue/libgeoclue.la $(GEOCLUE_LIBS)
 
 # This includes the standard gtk-doc make rules, copied by gtkdocize.
